src/Components/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../Utils/api";
+import { CommentCard } from "./CommentCard";
+
+export const CommentsList = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setLoading(true);
+
+    getCommentsByArticleId(article_id)
+      .then((comments) => {
+        setComments(comments);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.error("Error fetching comments:", error);
+        setLoading(false);
+      });
+  }, [article_id]);
+
+  if (loading) {
+    return <div>Loading comments...</div>;
+  }
+
```

Comment:
For an article where there are no comments (for example here: 'TenHag the man!') please display a message to the user, something like: 'No comments yet'.